apps/editing-toolkit/editing-toolkit-plugin/block-patterns/test/class-block-patterns-from-api-test.php

```diff
@@ -107,12 +107,42 @@ public function test_patterns_request_succeeds_with_empty_cache() {
 		$this->assertEquals( array( 'a8c/' . $this->pattern_mock_object['name'] => true ), $block_patterns_from_api->register_patterns() );
 	}
 
+	/**
+	 *  Tests that we're making two requests when we specify that we're on the site editor.
+	 */
+	public function test_patterns_site_editor_source_site() {
+		/**
+		 * A callback for the `a8c_enable_fse_block_patterns_api` filter.
+		 *
+		 * @return bool
+		 */
+		$should_enable_fse_block_patterns_api = function () {
+			return true;
+		};
+
+		add_filter( 'a8c_enable_fse_block_patterns_api', $should_enable_fse_block_patterns_api );
```

Comment:
Tiny nit, and totally optional, but could we reduce a few lines by using [__return_true](https://developer.wordpress.org/reference/functions/__return_true/) as the callback?

Reply:
Works for me! No harm in removing unnecessary lines :)